frontend/src/App.tsx

```diff
@@ -1,10 +1,20 @@
 import './App.css'
+import MainSearch from './components/MainSearch';
 
 function App() {
 
   return (
     <>
-      <h1>PeerNotes</h1>
+      <div className="App" style={{ display: 'flex', flexDirection: 'column', justifyContent: 'center', alignItems: 'center', height: '100vh' }}>
```

Comment:
We could add something like `gap: 1` here for a little bit of spacing between image and MainSearch. Let me know if you think that would look good 👍